model/AI/Classes/AIPackage.md

```diff
@@ -72,3 +70,34 @@ Metadata information that can be added to a package to describe an AI applicatio
   - type: SafetyRiskAssessmentType
   - minCount: 0
   - maxCount: 1
+
+## External properties restrictions
+
+- /Core/Properties/createdBy
```

Comment:
This should be referring to use in a Class, rather than Properties.     Consistent info that needs to be resolved in the rest of these.

Reply:
Thanks @maxhbr for fixing this up.